### DISCUSSION TITLE
v12 Drastic Load Reduction?

### DISCUSSION THREAD

**Discussion by leadZERO** [blakeblackshear/frigate#6397] [General]
Greetings! Just upgraded to v12 after being away for a few months. I noticed my server had a very drastic reduction in load averages with the new version. I hadn't followed all the changes and just used the release notes to update my config to go2rtc and remove RTMP.

I was just curious, is it go2rtc that caused such a big reduction in load?

![image](https://user-images.githubusercontent.com/1274368/236458099-3e57e50d-cac9-4cf8-9629-fdb27bd654b6.png)

**Comment by NickM-27** [comment]
It was not go2rtc that caused this, we had a number of optimizations around ffmpeg processes, hardware acceleration, etc.  to reduce CPU usage.
